NCR-Corporation/ncr-retail-demo
# Bug fix for /sites and cart problems

## Description

This includes a bug fix for:
1. The /sites page which was crashing with inactive sites being returned
2. Adding to cart because of a change in emerald urls

## Testing

Please outline how to test this fix or feature. Include all steps.

1. Visit `/sites` to confirm both sites load and it doesn't crash
2. Add items to cart and checkout to confirm flow still works
